assets/scripts/store/actions/map.js

```diff
@@ -0,0 +1,10 @@
+import { SET_MAP_STATE } from './index'
+
+const mapState = {}
```

Comment:
This is giving you a warning because you don't need it! `mapState` is defined as an argument to the function in line 5.